lib/Core/Executor.cpp

```diff
@@ -2566,16 +2576,92 @@ void Executor::executeInstruction(ExecutionState &state, KInstruction *ki) {
     break;
   }
 #endif
+  case Instruction::InsertElement: {
+    InsertElementInst *iei = cast<InsertElementInst>(i);
+    ref<Expr> vec = eval(ki, 0, state).value;
+    ref<Expr> newElt = eval(ki, 1, state).value;
```

Comment:
We can't do this. KLEE's expression language is completely untyped so `newElt` doesn't have a type. We could insert a check that does something like

```
assert(
  dyn_cast<VectorTy>(i->getOperand(0)->getType())->getElementType() == 
  i->getOperand(1)->getType());
```

This seems unnecessary though because I would expect this to hold by construction (i.e. I would expect LLVM's own asserts to enforce this).